### ISSUE TITLE
Food label claims (general/dietary/nutritional/etc.)

### ISSUE THREAD

**Issue by cmrn-rhi** [FoodOntology/joint-food-ontology-wg#26]
IFOW 2022 fermentation workshop highlighted the need to have a deeper dive into nutritional (and non-nutritional) label claims and thought we better capture this discussion point here before it slips away. FoodOn has an imported branch from LanguaL http://purl.obolibrary.org/obo/FOODON_03510032 that the FoodOn team was poking around last week. We've decided on some preliminary edits (like making the upper parent class term a new non-dietary use label claim - curation currently underway) but clearly this needs some more thought, modeling/exploration, and potential integration of existing claim standards.

![image](https://user-images.githubusercontent.com/48695054/184950005-8fc09558-decf-4b31-982d-9fd3bd5a221d.png)

**Comment by ddooley** [comment]
The "dietary use/label claim" term has been renamed "label claim or use".
However we were thinking of going one step further and renaming that "label claim" to stress that this is about what is written on - or the intention of - a label, rather than some scientific result of testing a product.  Feedback appreciated

**Comment by cmrn-rhi** [comment]
I agree, a "dietary use claim" seems like a subclass of "label claim".

**Comment by ddooley** [comment]
I've renamed all the underlying labels to remove the "or use" suffix since that isn't referenced in the definitions of the terms (where definitions exist).

**Comment by ddooley** [comment]
Dietary claim is now a subclass of label claim!
